src/api/tokens/types.d.ts

```diff
@@ -6,9 +6,12 @@
 
 export interface AccessToken {
   label: string
-  created: number
+  validUntil: string
+  keyId: string
+  createdAt: string
+  lastUsed: string
```

Comment:
The other date related properties had an adjective.

```suggestion
  lastUsedAt: string
```

Reply:
https://github.com/hedgedoc/hedgedoc/blob/develop/src/auth/auth-token.dto.ts#L19

Reply:
I'm crying